docs/_static/css/robots.txt

```diff
@@ -0,0 +1,82 @@
+Sitemap: http://docs.soliditylang.org/sitemap.xml
+
+# Prevent development and old documentation from showing up in search results.
+User-agent: *
+Disallow: /en/develop/
+Disallow: /en/breaking/
+
+# Disallow old versions
+Disallow: /en/v0.1.2/
+Disallow: /en/v0.1.3/
+Disallow: /en/v0.1.4/
+Disallow: /en/v0.1.5/
+Disallow: /en/v0.1.6/
+Disallow: /en/v0.1.7/
+Disallow: /en/v0.2.0/
+Disallow: /en/v0.2.1/
+Disallow: /en/v0.2.2/
+Disallow: /en/v0.3.0/
+Disallow: /en/v0.3.1/
+Disallow: /en/v0.3.2/
+Disallow: /en/v0.3.3/
+Disallow: /en/v0.3.4/
+Disallow: /en/v0.3.5/
+Disallow: /en/v0.3.6/
+Disallow: /en/v0.4.0/
+Disallow: /en/v0.4.1/
+Disallow: /en/v0.4.2/
+Disallow: /en/v0.4.3/
+Disallow: /en/v0.4.4/
+Disallow: /en/v0.4.5/
+Disallow: /en/v0.4.6/
+Disallow: /en/v0.4.7/
+Disallow: /en/v0.4.8/
+Disallow: /en/v0.4.9/
+Disallow: /en/v0.4.10/
+Disallow: /en/v0.4.11/
+Disallow: /en/v0.4.12/
+Disallow: /en/v0.4.13/
+Disallow: /en/v0.4.14/
+Disallow: /en/v0.4.15/
+Disallow: /en/v0.4.16/
+Disallow: /en/v0.4.17/
+Disallow: /en/v0.4.18/
+Disallow: /en/v0.4.19/
+Disallow: /en/v0.4.20/
+Disallow: /en/v0.4.21/
+Disallow: /en/v0.4.22/
+Disallow: /en/v0.4.23/
+Disallow: /en/v0.4.24/
+Disallow: /en/v0.4.25/
+Disallow: /en/v0.4.26/
+Disallow: /en/v0.5.0/
+Disallow: /en/v0.5.1/
+Disallow: /en/v0.5.2/
+Disallow: /en/v0.5.3/
+Disallow: /en/v0.5.4/
+Disallow: /en/v0.5.5/
+Disallow: /en/v0.5.6/
+Disallow: /en/v0.5.7/
+Disallow: /en/v0.5.8/
+Disallow: /en/v0.5.9/
+Disallow: /en/v0.5.10/
+Disallow: /en/v0.5.11/
+Disallow: /en/v0.5.12/
+Disallow: /en/v0.5.13/
+Disallow: /en/v0.5.14/
+Disallow: /en/v0.5.15/
+Disallow: /en/v0.5.16/
+Disallow: /en/v0.5.17/
+Disallow: /en/v0.6.0/
+Disallow: /en/v0.6.1/
+Disallow: /en/v0.6.2/
+Disallow: /en/v0.6.3/
+Disallow: /en/v0.6.4/
+Disallow: /en/v0.6.5/
+Disallow: /en/v0.6.6/
+Disallow: /en/v0.6.7/
+Disallow: /en/v0.6.8/
+Disallow: /en/v0.6.9/
+Disallow: /en/v0.6.10/
+Disallow: /en/v0.6.11/
+Disallow: /en/v0.6.12/
```

Comment:
I don't think we need all the 0.7.x and 0.8.x minor releases. The latest one should be enough.

Thought in that case we should probably have a rule in the release checklist to add blacklist the previous version here on release.